lib/croods/controller/collection.rb

```diff
@@ -5,10 +5,22 @@ class Controller < ActionController::API
     module Collection
       protected
 
+      def page_param
```

Comment:
@danielweinmann Should these 2 methods live here or inside Resource::Pagination?

Reply:
I like them here